frontend/package.json

```
@@ -6,13 +6,15 @@
   "scripts": {
     "build": "remix vite:build",
     "dev": "remix vite:dev --port 5101",
-    "preview": "wrangler pages dev ./build/client --port=5100 --binding SCRIPT_FRONTEND_URL=http://localhost:5100/ --compatibility-date=2024-09-23 --compatibility-flag=nodejs_compat"
+    "preview": "pnpm run build && wrangler pages dev ./build/client",
```

```suggestion
    "preview": "pnpm run build && wrangler pages dev",
```

And add to `wrangler.toml`:
```toml
pages_build_output_dir="./build/client"
```